src/platform/web/ui/RootView.js

```diff
@@ -19,7 +19,7 @@ import {LoginView} from "./login/LoginView.js";
 import {SessionLoadView} from "./login/SessionLoadView.js";
 import {SessionPickerView} from "./login/SessionPickerView.js";
 import {TemplateView} from "./general/TemplateView";
-import {StaticView} from "./general/StaticView.js";
+import {StaticView} from "./general/StaticView.ts";
```

Comment:
For typescript, we leave the extension off
```suggestion
import {StaticView} from "./general/StaticView";
```